### PR TITLE
feat: cli create test command

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR adds the command `tracetest test run --definition file.yml`. Right now it just creates the test if it doesn't exist yet. I'll open more PRs to add the functionality for waiting until the result is finished and to update an existing test.

Other than the run command itself, it also introduces the text format specified in #160.

Loom video explaining it: https://www.loom.com/share/f738c2c1ef8e4500a358b76fa7ad1e63

## Checklist

- [x] tested locally
- [x] added new dependencies
- [ ] updated the docs
- [x] added a test

### CODE REVIEW

File: api/http.yaml [hunk 1]
```diff
@@ -23,7 +23,6 @@ components:
             $ref: "#/components/schemas/HTTPHeader"
         body:
           type: string
-          format: byte
```
Comment (mathnogueira) [inline api/http.yaml:26]: Basically, this means the `body` should be in `base64` format. But this is not true. I removed this so prism could add the correct validations to the requests I'm sending.

File: cli/testdata/definitions/valid_http_test_definition.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - messaging.message.payload contains 52
+- selector: span[span.name = "consume message from queue"]:last
+  assertions:
+  - messaging.message.payload contains 52
+- selector: span[span.name = "consume message from queue"]:last span[span.name = "import pokemon from pokeapi"]
+  assertions:
+  - http.status = 200
+- selector: span[span.name = "consume message from queue"]:last span[span.name = "save pokemon on database"]
+  assertions:
+  - db.repository.operation = "create"
```
Comment (mathnogueira) [inline cli/testdata/definitions/valid_http_test_definition.yml:34]: There are some details in the syntax here. When defining an assertion, the value must be quoted if it is a string. Right now we support `int`, `double`, and `string` values.

File: api/http.yaml (hunk 1 above)
Comment (schoren) [reply api/http.yaml:26]: good catch

File: cli/testdata/definitions/valid_http_test_definition.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+name: POST import pokemon
+description: Import a pokemon using its ID
```
Comment (schoren) [inline cli/testdata/definitions/valid_http_test_definition.yml:2]: do we have a `description` in the UI?

File: cli/file/definition.go [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	"os"
+
+	"github.com/kubeshop/tracetest/cli/definition"
+	"gopkg.in/yaml.v2"
+)
+
+func LoadDefinition(file string) (definition.Test, error) {
+	fileBytes, err := os.ReadFile(file)
+	if err != nil {
+		return definition.Test{}, fmt.Errorf("could not read file %s: %w", file, err)
```
Comment (schoren) [inline cli/file/definition.go:14]: nit: a more specific message could help debugging, like `could not read test definition file...`

File: cli/conversion/parser/assertion_parser.go [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+	}
+
+	var assertionParserObject assertionParserObject
+	err = parser.ParseString("", assertionQuery, &assertionParserObject)
+	if err != nil {
+		return Assertion{}, fmt.Errorf("could not parse assertion (%s): %w", assertionQuery, err)
+	}
+
+	value := assertionParserObject.Value
+	if value[0:1] == "\"" {
```
Comment (schoren) [inline cli/conversion/parser/assertion_parser.go:60]: maybe this looks easier to read?
```suggestion
	if value[0:1] == `"` {
```

**Comment by mathnogueira** [comment]
Applied all changes! Thanks for the review @schoren

**Pull request merged**